src/statistics.f90

```diff
@@ -304,6 +304,7 @@ subroutine overall_statistic(ux1,uy1,uz1,phi1,pp3,ep1)
        call restart_statistic()
     endif
 
+    if (mod(itime,istatfreq)/=0) return
```

Comment:
I think the latter, starting from `time=initstat` this would sample every `istatfreq`'th timestep whereas the former would be every `istatfreq`'th timestep starting from `t=0`.

I suppose in terms of converged statistics it shouldn't matter, however I think most people will expect sampling at a given rate starting from `initstat`.

Reply:
Yes I think that makes sense

Reply:
I have updated it and also changed it so the statistics will output to file if the statistics are not computed